models/cm_compartmentcurrents.h

```diff
@@ -466,6 +468,8 @@ class CompartmentCurrents
     {
       assert( false );
     }
+
+    ALL_ENTRIES_ACCESSED( *receptor_params, "receptor_params", "Unread dictionary entries: " );
```

Comment:
The dict access checking looks good to me.

Reply:
Ok, thx!